src/components/views/auth/InteractiveAuthEntryComponents.tsx

```diff
@@ -621,6 +621,94 @@ export class MsisdnAuthEntry extends React.Component<IMsisdnAuthEntryProps, IMsi
     }
 }
 
+interface IRegistrationTokenAuthEntryState {
+    registrationToken: string;
+}
+
+@replaceableComponent("views.auth.RegistrationTokenAuthEntry")
+export class RegistrationTokenAuthEntry extends React.Component<IAuthEntryProps, IRegistrationTokenAuthEntryState> {
+    static LOGIN_TYPE = AuthType.RegistrationToken;
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            registrationToken: "",
```

Comment:
Why an empty string and not null?

Reply:
Changed to null, think I was just copying the empty string from the password section. Out of interest, why is null preferable to an empty string?